packages/flutter_driver/lib/src/driver.dart

```diff
@@ -211,15 +211,21 @@ class FlutterDriver {
     return await _sendCommand(new Scroll(finder, dx, dy, duration, frequency)).then((Map<String, dynamic> _) => null);
   }
 
+  /// Tell the driver to ensure that the widget located by [finder] has been
+  /// scrolled completely into view. See [Scrollable.ensureVisible].
```

Comment:
How about simply "Scrolls the widget located by [finder] into view so that it is completely visible"?